Add 2 missing dependencies of dependencies

natefoo opened the pull request:
As a nice benefit of using nearly standard tooling, Galaxy still started fine without these being explicitly defined and placed out on the wheels server. Previously the `requests-toolbelt` wheel would be downloaded from PyPI, and the `Markdown` sdist (which is also pure Python but only has an sdist in PyPI) would be pulled from PyPI and built/installed by pip.

natefoo committed:
Add 2 missing dependencies of dependencies

nsoranzo commented:
It seems that kombu-3.0.26 on Python 2.6 requires `importlib` and `ordereddict`:

```
Collecting importlib (from kombu==3.0.26->-r requirements.txt (line 39))
  Downloading importlib-1.0.3.tar.bz2
Collecting ordereddict (from kombu==3.0.26->-r requirements.txt (line 39))
  Downloading ordereddict-1.1.tar.gz
Building wheels for collected packages: importlib, ordereddict
  Running setup.py bdist_wheel for importlib
  Stored in directory: /home/soranzon/.cache/pip/wheels/06/7c/8c/89d4e783fed7720b8f4a0c221e7c46d0557842bfca5c96aefb
  Running setup.py bdist_wheel for ordereddict
  Stored in directory: /home/soranzon/.cache/pip/wheels/4e/23/88/f33a8c7c622e6f5244c408306805de7c56dd5d36cff4346fb6
Successfully built importlib ordereddict
```

natefoo committed:
Add importlib and ordereddict for Python 2.6.

They won't be installed by conditional since they will always be
installed as dependencies of kombu in requirements.txt. This should be
fixed at some point.

natefoo committed:
XPath support is not the same on all Python 2.6es

natefoo commented:
bump :punch:

nsoranzo commented:
:+1:
